README.md

```diff
@@ -38,6 +38,7 @@ Different backends are available see:
 - Pantheon (Elementary) [xdg-desktop-portal-pantheon](https://github.com/elementary/portals)
 - wlroots [xdg-desktop-portal-wlr](https://github.com/emersion/xdg-desktop-portal-wlr)
 - Deepin [xdg-desktop-portal-dde](https://github.com/linuxdeepin/xdg-desktop-portal-dde)
+- Cinnamon [xdg-desktop-portal-xapp](https://github.com/linuxmint/xdg-desktop-portal-xapp)
```

Comment:
Calling it the Xapp portal would probably be more correct instead of Cinnamon since XFCE and MATE are supported in the same codebase.

Reply:
Fixed by using Simon's wording. Thanks!